tests/dummy/app/controllers/editable-node.ts

```diff
@@ -83,23 +84,28 @@ export default class EditableBlockController extends Controller {
   @service declare intl: IntlService;
   schema = new Schema({
     nodes: {
-      doc: docWithConfig({
-        defaultLanguage: 'nl-BE',
-      }),
+      doc: {
+        ...doc({
+          defaultLanguage: 'nl-BE',
+          rdfaAware: true,
+        }),
+        ...sharedRdfaNodeSpec,
```

Comment:
This doesn't feel like the right way to do this. It requires anyone who wants to start using the `rdfaAware` features to know that they also need to do this for some of the nodes. To me, having a `config` option implies that switching that option should take care of turning on the feature rather than being part of the picture. I think spreading (or not) the shared nodespec should be done in the specs themselves.

Reply:
That's fair, I'm wondering if we should automatically enable 'editable'/'isolating'/'selectable' when `rdfaAware` is enabled, or if we should add additional 'opt-in' options for these to the `Config` object.

Reply:
As (at the moment at least), we do not always want the new rdfa-aware nodes to be editable/isolating/selectable.

Reply:
Is it something that the (dev) users are going to want to alter, or is it always going to be the same for a NodeSpec?

To me it seems most likely that while some NodeSpecs want different settings, they are like that for a reason, so we don't really want (dev) users playing with them as it's cause weird problems.

If we do want to let users play with them, then I think setting sensible defaults when they pass `rdfaAware` but then allowing them to be set explicitly with other config options, is the way to go.

Reply:
I guess we can indeed turn editable/selectable/isolating on by default (when `rdfaAware` is enabled and where applicable), dev users can indeed override them.
It would turn it on for:
- block_rdfa
- inline_rdfa
- heading?
- doc